string id
float32 x
float32 y
float32 coleta_x     # novo
float32 coleta_y     # novo
float32 dest_x
float32 dest_y
string status
string robot_id
float64 timestamp